openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AddAndRemoveTierAndOwner.spec.js

```diff
@@ -41,28 +41,55 @@ const ENTITIES = {
 const glossary = 'GlossaryOwnerTest';
 const glossaryTerm = 'GlossaryTermOwnerTest';
 
-const OWNER = 'Aaron Singh';
+const OWNER = 'Amber Green';
```

Comment:
why we are changing the name?

Reply:
while testing on the upgrade tests, there were multiple users named Aaron Singh.